Dockerfile

```diff
@@ -85,6 +85,7 @@ EXPOSE 5000
 # Run server
 # exec is used to make sure signals are propagated to uwsgi,
 # while also allowing shell expansion
+SHELL ["/bin/bash", "-c"]
```

Comment:
Could you move the `SHELL` directive into its own paragraph, perhaps somewhere at the top of the `Dockerfile`?  E.g. in the Ntupling Service backend we are using:

```
# Configure shell options
SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```